### PR TITLE
add Cascadia chain in testnet category

### PR DESCRIPTION
**Pull request by ryssroad**

### CODE REVIEW

File: testnets/cascadiatestnet/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        {
+          "denom": "TCC",
+          "exponent": 0
+        },
+        {
+          "denom": "aCC",
+          "exponent": 18
+        }
+      ],
+      "base": "cascadia",
```
Comment (JeremyParish69) [inline testnets/cascadiatestnet/assetlist.json:17]: the base unit must also be defined in denom_units array

File: testnets/cascadiatestnet/chain.json [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  "daemon_name": "cascadiad",
+  "node_home": "$HOME/.cascadiad",
+  "key_algos": [
+    "ethsecp256k1"
+  ],
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "aCC",
```
Comment (JeremyParish69) [inline testnets/cascadiatestnet/chain.json:19]: the fee token must be the base unit, but it looks like aCC is the 18 exponent unit. Should this be TCC?

Comment (ryssroad) [reply testnets/cascadiatestnet/chain.json:19]: fixed above

File: testnets/cascadiatestnet/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      "name": "TCC",
+      "display": "TCC",
```
[the remark this answers is not in the document]
Comment (ryssroad) [reply testnets/cascadiatestnet/assetlist.json]: fixed, recheck pls

File: testnets/cascadiatestnet/chain.json [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+        "low_gas_price": 0.05,
+        "average_gas_price": 0.05,
+        "high_gas_price": 0.1
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "tCC"
```
Comment (JeremyParish69) [inline testnets/cascadiatestnet/chain.json:30]: The correction you've implemented have made it much more clear which is the base unit of this token. The staking token must also refer to a base denom, so this should be aCC instead?

Comment (ryssroad) [reply testnets/cascadiatestnet/chain.json]: yeah, aCC, fixed

**Pull request merged**